### ISSUE TITLE
Does the `metrics` actually work?

### ISSUE THREAD

**Issue by FUSAKLA** [PostHog/charts#17]
I see the chart allows to deploy the prometheus statsd exporter but I do not see any data there and it looks like PostHog actually does not support statsd metrics at all. 
Or am I missing something? 

Thanks in advance!

**Comment by mariusandra** [comment]
@fuziontech is this something you could investigate? I didn't put a lot of emphasis on metrics when making this chart unfortunately.

@FUSAKLA any PR from your side for what you need this chart to support is more than welcome!

**Comment by fuziontech** [comment]
This is something I'd really like to get into Posthog more is metrics in general. It's definitely on the roadmap. Once we have some basic metrics in place I'll be sure to comment here.

**Comment by FUSAKLA** [comment]
Great, I looked at the code and could not find any metrics references.
I saw that Django should support some statsd middle-ware which could be used  same as Prometheus middle-ware.

If I find a time I'll send a PR, but I'd say this needs bit more discussion from maintainers which direction of monitoring tools they would like to go.

**Comment by yakkomajuri** [comment]
I think this can be closed at this point? @fuziontech

**Comment by fuziontech** [comment]
not quite, we have metrics in statsd but I think giving some attention to Prometheus would be good so that we can incorporate this into the chart
